pkgs/top-level/python-packages.nix

```diff
@@ -8751,6 +8751,8 @@ in {
     };
   };
 
+  pepper = callPackage ../development/python-modules/pepper { };
```

Comment:
@FRidh I have eventually opted for pkgs/tools/admin/salt/pepper which seems good enough. I have also replaced `buildPythonPackage` with `buildPythonApplication`. 
As a note, I don't know how to find out the proper `mirror://pypi/` (but `github` releases works well enough).